code/modules/mob/living/carbon/life.dm

```diff
@@ -120,6 +118,11 @@
 
 	check_breath(breath)
 
+	if(istype(loc, /obj/machinery/atmospherics/components/unary/cryo_cell))
+		var/obj/machinery/atmospherics/components/unary/cryo_cell/loc_as_cryo = loc
+		loc_as_cryo.airs[1].merge(breath)
+		return
+
 	if(breath)
 		loc.assume_air(breath)
```

Comment:
why are you doing this here?

Reply:
so that the gases that the mob inside breathes out are not put in the atmosphere but are recycled inside as a closed system.

Reply:
couldn't you do this in cryo_cell/assume_air?